tinodesdk/src/main/java/co/tinode/tinodesdk/Topic.java

```diff
@@ -454,8 +454,15 @@ protected void setSeqAndFetch(int seq) {
             if (!isAttached()) {
                 try {
                     // Fully asynchronous fire and forget.
```

Comment:
Please remove the comment because it no longer applies.

Reply:
done.